tests/integration/test_charm_fork_repo.py

```diff
@@ -174,9 +177,43 @@ async def test_dispatch_workflow_failure(
             break
         sleep(30)
     else:
-        assert False, "Timeout wait for workflow to complete"
+        assert False, "Timeout while waiting for workflow to complete"
 
     # The only job in the workflow is job that `echo` the runner name. If it fails then it should
     # be the pre-run job that failed.
     run = workflow.get_runs()[0]
     assert run.jobs()[0].conclusion == "failure"
+
+
+@pytest.mark.asyncio
+@pytest.mark.abort_on_fail
+async def test_path_config_change(
+    model: Model,
+    app_with_unsigned_commit_repo: Application,
```

Comment:
Is this fixture name still accurate?

Reply:
This test requires two repo to be setup. Currently, this is reusing the fork repo with unsigned commit, hence the name of the fixture used.
I felt create a new forked repo just for this one test was overboard.